app/dao/daily_sorted_letter_dao.py

```diff
@@ -35,4 +35,4 @@ def dao_create_or_update_daily_sorted_letter(new_daily_sorted_letter):
             'updated_at': datetime.utcnow()
         }
     )
-    db.session.connection().execute(stmt)
+    db.session.using_bind('writer').connection().execute(stmt)
```

Comment:
All custom SQL statements need the explicit bind because the `flushing` property will be false and the SQL clause will be set to `None`, hence there will be no way (that I know of at the moment) to detect automatically if this query will modify the database's state.